### DISCUSSION TITLE
为啥不支持代码的样式展示？

### DISCUSSION THREAD

**Discussion by xnzxchun** [dirk1983/chatgpt#69] [Q&A]
和作者的样式不一样
![image](https://user-images.githubusercontent.com/128781530/227441661-695aa154-aacd-43cd-a7b5-a3b05b69f2b8.png)

**Comment by dirk1983** [comment]
因为返回的代码并没有放到···标签里面，也就是说不是markdown格式的，所以不会解析成代码。

**Reply by xnzxchun** [reply]
好的
